src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueueModule.scala

```diff
@@ -43,6 +50,18 @@ class LaunchQueueModule(
 
   val launchQueue: LaunchQueue = new LaunchQueueDelegate(config, launchQueueActorRef, rateLimiterActor)
 
+  // TODO(jdef) copied this from auth.AuthModule; seems useful to have this as a reusable func
```

Comment:
so it'd be really cool if this func combined all the plugins that satisfied an Opt.Factory[_,_] interface into a single factory via `combine`